src/CMakeLists.txt

```diff
@@ -138,6 +138,9 @@ libcvc4_add_sources(
   proof/lean/lean_post_processor.cpp
   proof/lean/lean_post_processor.h
   proof/lean/lean_printer.h
+  proof/veriT/veriT_post_processor.cpp
```

Comment:
As a style comment, I think we should use `verit` not `veriT` here.

Reply:
Agreed. The directory should also be called `verit`.

Reply:
Done